core/src/main/java/org/teavm/libgdx/emu/PixmapEmulator.java

```diff
@@ -26,10 +26,10 @@
 import java.nio.IntBuffer;
 import java.util.HashMap;
 import java.util.Map;
-import org.teavm.dom.browser.Window;
+import org.teavm.jso.browser.Window;
 import org.teavm.dom.canvas.CanvasRenderingContext2D;
```

Comment:
I can see sevarl `teavm.dom` imports (instead of `teavm.jso.dom`). Why did not you update them as well? Does this code compile?

Reply:
@konsoletyper This code does not compile yet. I'm still working on it. Hence the "WIP" in the title of the pull request. Sorry, if this was misleading. :-)

Reply:
@konsoletyper In any case, thank you for creating this project and for your fast reaction on creating this pull request. I'll ping you once I think it is ready if you don't mind.